### PR TITLE
Update ta-IN.json

### PR DESCRIPTION
**Pull request by arunkarthicknm**

### CODE REVIEW

**Review by guyluz11 — COMMENTED** [review]
It looks like previous people created some duplicate translations.

I see that you have already removed `Move_to_blinds_full_settings_page` but there is more like `Scene_Settings_Page`, and `Welcome_to_your_Smart_Home`.

Can you find all of the duplicates and remove them?.

Please keep only the onces in the original line number as the one in [en-US.json](https://github.com/CyBear-Jinni/cbj_app/blob/dev/assets/translations/en-US.json) page

**Comment by arunkarthicknm** [comment]
Fine now? @guyluz11 :)

File: assets/translations/ta-IN.json [hunk 1]
```diff
@@ -17,7 +17,6 @@
   "Lamps_Settings_Page": "விளக்குகள் அமைப்புகள் பக்கம்",
   "Welcome_to_your_Smart_Home": "உங்கள் ஸ்மார்ட் ஹோமுக்கு வரவேற்கிறோம்",
   "Scene_Settings_Page": "காட்சி அமைப்புகள் பக்கம்",
```
Comment (guyluz11) [inline assets/translations/ta-IN.json:19]: Remove this line it is already exists

File: assets/translations/ta-IN.json [hunk 2]
```diff
@@ -17,7 +17,6 @@
   "Lamps_Settings_Page": "விளக்குகள் அமைப்புகள் பக்கம்",
   "Welcome_to_your_Smart_Home": "உங்கள் ஸ்மார்ட் ஹோமுக்கு வரவேற்கிறோம்",
```
Comment (guyluz11) [inline assets/translations/ta-IN.json:18]: Remove this line it is already exists

File: assets/translations/ta-IN.json [hunk 3]
```diff
@@ -17,7 +17,6 @@
   "Lamps_Settings_Page": "விளக்குகள் அமைப்புகள் பக்கம்",
```
Comment (guyluz11) [inline assets/translations/ta-IN.json:17]: Remove this line it is already exists

File: assets/translations/ta-IN.json [hunk 4]
```diff
@@ -17,7 +17,6 @@
...
   "Welcome_to_your_Smart_Home": "உங்கள் ஸ்மார்ட் ஹோமுக்கு வரவேற்கிறோம்",
   "Scene_Settings_Page": "காட்சி அமைப்புகள் பக்கம்",
-  "Move_to_blinds_full_settings_page": "Move to blinds full settings page",
   "Blinds_Page": "குருட்டுப் பக்கம்",
```
Comment (guyluz11) [inline assets/translations/ta-IN.json:20]: Remove this line it is already exists

**Review by guyluz11 — CHANGES_REQUESTED** [review]
Plase remove all duplicates, I see that there are more like

`History_Settings_Page` `Name:_` `Room:_` `Room:_`

**Review by arunkarthicknm — COMMENTED** [review]
Now?

**Review by guyluz11 — APPROVED** [review]
Looks good :)

**Comment by guyluz11** [comment]
@arunkarthicknm thanks for the pr and a happy Hacktoberfest 🤠

**Pull request merged**